controllers/constants/constants.go

```diff
@@ -18,8 +18,6 @@ package constants
 const (
 	InferenceServiceKind = "InferenceService"
 
-	IstioNamespace                   = "istio-system"
```

Comment:
Note that it is still used in case the func `utils.GetIstioControlPlaneName`

can't find anything: https://github.com/mholder6/odh-model-controller/blob/7a14b2092913e57cc2367ce14c0fa7787b2b6d1e/controllers/utils/utils.go#L217C6-L217C46

same for `IstioControlPlaneName`

It can't be deleted, you need to override where these values are hardcoded to the values returned from this function.

Reply:
Yes I actually am invoking panic instead of defaulting to those values -- you can see this here:
https://github.com/opendatahub-io/odh-model-controller/pull/254/files#diff-d93cfa921e60384e543609474ebc28fdbad7ae20f135f76398ef8d821333b3f2R194